sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -46,11 +46,13 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
   private val df = ds.toDF()
 
   /**
+   * <ul>
    * Specifies the behavior when data or table already exists. Options include:
```

Comment:
This one looks wrongly formatted btw. Looks ul should be below of this line

Reply:
Crap yeah, will move it down